travis_after_script.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+if [ "$TRAVIS_PULL_REQUEST" = "false" ]
+then
+docker exec openmrs-referenceapplication cat .OpenMRS/openmrs.log
```

Comment:
Could we please echo "\n\nLoading .OpenMRS/openmrs.log from the server:\n\n" before printing out logs?